docs/installation.rst

```diff
@@ -5,6 +5,26 @@ Installation
 .. note::
     This page documents how to install FAME on Ubuntu 20.04. FAME being written in Python, you can install it on the system of your choice.
 
+The easy way
+============
+
+Probably the most easy way to run fame is::
+
+    $ git clone https://github.com/certsocietegenerale/fame.git
+    $ cd fame/docker
+    $ cp fame.env.template fame.env
+    $ sed -i "s/#privileged: true/privileged: true/g" docker-compose.yml
+    $ docker-compose up --build
+
+
+Then browse http://localhost:4200/ using your web browser, and login using the `docker first run default credentials <https://github.com/certsocietegenerale/fame/blob/master/docker/fame.env.template>`_  (admin@changeme.fame / changeme)
+
+These commands are very nice for having a first look to what fame is, but you may also want to install fame without docker.
+
+This is especially true given that FAME use docker internally for its activities, which leads to Docker-in-Docker situation. DinD is dangerous, we do not recommend running FAME in docker in production.
```

Comment:
I'm not sure this is true. Is it really more dangerous than a typical installation where the fame users is in the docker users ?

Reply:
You are absolutely right.
I wrote this as an abundance of caution : many people have a false sense that "docker containers are like virtual machines : they are isolated sandboxes". This is true to some extend but...container evasion is a lot more easier than VM evasion. Especially when the container is privileged.

I wrote this disclaimer because I wanted to clarify that fame worker really need to be on its dedicated server. I will reword this to avoid spreading FUD